inspire_dojson/processors.py

```diff
@@ -58,6 +59,17 @@ def overdo_marc_dict(record):
         return hep.do(record)
 
 
+def json2marcxml(record):
+    """Convert INSPIRE JSON into MARC Groupable Ordered Dict."""
+    schema_name = _get_schema_name_from_record(record)
+    if schema_name == 'hep':
+        return hep2marc.do(record)
+    elif schema_name == 'authors':
+        return hepnames2marc.do(record)
+    else:
+        raise RuntimeError(u"Conversion from JSON to MARCXML is not supported for the schema {}".format(schema_name))
```

Comment:
`ValueError`?

Reply:
The thing is that this depends typically on input that is passed directly from client code. So e.g. if one will ask a MARCXML export of an experiment, it's not the experiment *per se* that is wrong (i.e. the value), but that we are not yet ready to convert that. Maybe I should rather use `NotImplementedError`.